static/app/views/insights/browser/webVitals/components/tables/pagePerformanceTable.tsx

```diff
@@ -67,6 +71,9 @@ export function PagePerformanceTable() {
 
   const query = decodeScalar(location.query.query, '');
   const browserTypes = decodeBrowserTypes(location.query[SpanIndexedField.BROWSER_NAME]);
+  const subregions = decodeList(
```

Comment:
I think it would make sense to use a custom query decoder for all of these places we're decoding the subregions -- similar to `decodeBrowserTypes` -- so that we can guarantee the regions are valid

Reply:
Hmm, When would this occur?

Reply:
I'm going to address this in another PR, i'm still not 100% sure on the benefit so i'll give it some thought, there are 2 cases I can see that would produce an invalid regions.

1. Our code somehow produces an invalid subregion. In this case, i feel as though adding decodeSubregion might actually mask any problem we have in code that shouldn't be occurring anyways. I think we would want to see if this problem occurs. 
2. The user manually enters a subregion in the url bar. In this case the query should succeed here anyways, unless they enter a string that happens to match some other query syntax. Which in that case i think it might be better to show a failed query in our UI, so that the user knows what they entered in the url bar doesn't make sense. Either way I like the idea of showing the user the results of their own changes in the url bar.